src/scheduled_tasks.rs

```diff
@@ -32,9 +32,6 @@ pub fn setup(db_url: String, user_agent: String) -> Result<(), LemmyError> {
   let mut scheduler = Scheduler::new();
 
   let mut conn_1 = PgConnection::establish(&db_url).expect("could not establish connection");
```

Comment:
It seems like this connection will be kept open forever without being used. Better to move it and the functions which use it into a code block or a separate function.